src/main/java/com/commercetools/helper/mapper/impl/payment/BasePaymentMapperImpl.java

```diff
@@ -253,13 +254,21 @@ protected Details getTransactionDetails(@Nonnull CtpPaymentWithCart paymentWithC
      * @see #mapCustomLineItemToPaypalPlusItem(CustomLineItem, List)
      */
     protected Stream<Item> mapCustomLineItemToPaypalPlusItem(@Nonnull CustomLineItem customLineItem, @Nonnull List<Locale> locales) {
+        String itemName = customLineItem.getName().get(locales);
+        if (StringUtils.isBlank(itemName)) {
+            Locale fallBackLocale = customLineItem.getName().getLocales()
+                    .iterator().next();
+            itemName = customLineItem.getName().get(fallBackLocale);
```

Comment:
Can fallBackLocale be null here?

Reply:
I have modified to handle the null. Also from LocalizedString class will gurantee the iterator will not be null https://github.com/commercetools/commercetools-jvm-sdk/blob/cfa043b25ebec1656311643ca4201ec81a172fe2/commercetools-sdk-base/src/main/java/io/sphere/sdk/models/LocalizedString.java#L312